plugin/src/test/java/org/opensearch/ml/utils/TestHelper.java

```diff
@@ -309,6 +310,22 @@ public static RestRequest getMetricsCorrelationRestRequest() {
             .build();
     }
 
+    public static RestRequest getExecuteAgentRestRequest() {
+        Map<String, String> params = new HashMap<>();
+        params.put(PARAMETER_AGENT_ID, "test_agent_id");
+        final String requestContent = "{\"name\":\"Test_Agent_For_RAG\",\"type\":\"flow\","
```

Comment:
is this name accurate? this looks like using catIndexTool. Maybe rename to 'Test_Agent_For_CatIndex' ?